WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/interval/intervaltypes/TypingInterval.java

```diff
@@ -56,10 +76,25 @@ public void run() {
 	 */
 	@SerializedName("diff")
 	JsonifiedLong editDistance;
+	
+	/**
+	 * The number of characters that have been added, removed or modified during
+	 * this interval.
+	 */	
+	@SerializedName("modCount")
+	private int modCount;
+	
+	/**
+	 * The difference in the number of characters between the starting and ending
+	 * document.
+	 */
+	@SerializedName("startEndDiff")
```

Comment:
Not anymore. This serialized named belonged to the previous variable name, which was much longer. I'll fix this